packages/expo-router/assets/native-tabs.module.css

```diff
@@ -0,0 +1,114 @@
+.nativeTabsContainer {
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  max-height: 100vh;
+  max-width: 100vw;
+  overflow: hidden;
+}
+
+.tabContent {
+  flex: 1;
+  display: flex;
+  max-height: 100%;
+  max-width: 100%;
+}
+.tabContent[data-state="inactive"] {
+  display: none;
+}
+
+.navigationMenuRoot {
+  top: 24px;
+  left: 50%;
+  transform: translateX(-50%);
+  position: fixed;
```

Comment:
This won’t support nesting tabs. Wonder if it’ll support body scrolling in mobile web.

Reply:
Currently we don't need support for nested tabs. They don't work on Android and iOS

Reply:
Writing the support now will complicate the css here. I would wait for feedback from users, if anyone even needs the nested tabs